src/config/constants.ts

```diff
@@ -1,4 +1,5 @@
 export enum SupportedChainId {
   GOERLI = '5',
   GNOSIS_CHAIN = '100',
+  SEPOLIA = '11155111',
```

Comment:
We are missing the mapping to the respective API key in `src/config/configuration.ts`

https://github.com/safe-global/safe-gelato-relay-service/blob/d9244885b7b80a784d8f8d6105b84003aba7eea4/src/config/configuration.ts#L16-L21

Reply:
Thanks, added!